### ISSUE TITLE
Remove obselete CSS Selectors

### ISSUE THREAD

**Issue by ntwb** [WordPress/twentytwenty#627]
The following selectors are obsolete and should be removed:

The following are based on the stylelint-a11y [no-obsolete-attribute](https://github.com/YozhikM/stylelint-a11y/blob/master/src/rules/no-obsolete-attribute/README.md) rule:

Via https://html.spec.whatwg.org/multipage/obsolete.html#non-conforming-features

> 15.2 Non-conforming features
> 
> > Elements in the following list are entirely obsolete, and must not be used by authors:
>
> `big`
> 
> > Similarly, if the `big` element is being used to denote a heading, consider using the `h1` element; if it is being used for marking up important passages, consider the `strong` element; and if it is being used for highlighting text for reference purposes, consider the `mark` element.
>
> `acronym`
> >
> >  Use `abbr` instead.
>
> `strike`
> >
> > Use `del` instead if the element is marking an edit, otherwise use `s` instead.

https://github.com/WordPress/twentytwenty/blob/master/style.css#L71

https://github.com/WordPress/twentytwenty/blob/master/style.css#L78

https://github.com/WordPress/twentytwenty/blob/master/style.css#L364-L366

https://github.com/WordPress/twentytwenty/blob/master/style.css#L397-L400

**Comment by joyously** [comment]
Does this consider legacy content? (WordPress has been around 16 years.)
Also, if the styles are removed, we're back to issue #6. If you use CSS normalize instead of CSS reset, the browsers will take care of the legacy, deprecated tags so you don't have to.

**Comment by ntwb** [comment]
Great Q's, in this case it doesn't matter the CSS selectors are no longer used by the browsers in use today, they ignore all the CSS selectors mentioned, they are essentially _dead legacy_ code.

**Comment by joyously** [comment]
I don't think you have totally understood what I said, or what has been done already.
The browsers don't ignore those selectors, so they are not dead code. 
There could be lots of legacy sites in the world with those deprecated HTML tags. The browser will handle them fine unless the theme styles them differently. This theme uses CSS reset, so that is styling them when it shouldn't be (unless removed from the list for the reset). Other styles were added in previous PRs because I wrote an issue that they were unstyled. But I also wrote an issue to use CSS normalize instead of CSS reset, which would work much better for deprecated tags.
So it's not copy/paste. And it's not best practice to use CSS reset.

**Comment by glauberm** [comment]
I agree with @joyously . They shouldn't exist on the theme's HTML, but the CSS should handle its possible existence.

**Comment by joedolson** [comment]
Letting these elements remain styled in CSS will have no significant accessibility consequences. The accessibility concern is with using the elements, and the theme can't control that in content; they should still be styled appropriately. It's up to the content author to remove or replace the elements in their content. 

As a note, the stylelint-a11y rule is about attributes, not about elements, and doesn't directly apply to the case you're describing.
